ash/src/instance.rs

```diff
@@ -87,25 +87,30 @@ impl<V: FunctionPointers> Instance<V> {
 pub trait InstanceV1_1: InstanceV1_0 {
     fn fp_v1_1(&self) -> &vk::InstanceFnV1_1;
 
-    fn enumerate_physical_device_groups(&self) -> VkResult<Vec<vk::PhysicalDeviceGroupProperties>> {
```

Comment:
Can you remind me why this function needs to be changed? What is wrong with returning `Vec<vk::PhysicalDeviceGroupProperties>`? I think the user can just check the length to see if it would be `INCOMPLETE`.